operator/src/main/java/org/keycloak/operator/controllers/KeycloakDeploymentDependentResource.java

```diff
@@ -557,6 +558,12 @@ private List<EnvVar> getDefaultAndAdditionalEnvVars(Keycloak keycloakCR) {
         envVars.add(new EnvVarBuilder().withName(POD_IP).withNewValueFrom().withNewFieldRef()
                 .withFieldPath("status.podIP").withApiVersion("v1").endFieldRef().endValueFrom().build());
 
+        // Both status.hostIP or spec.nodeName would be fine here.
+        // In theory, status.hostIP is a smaller value and, as this value is tagged in all JGroups messages, it should have a lower overhead.
+        // Note: the value (IP) is shown in the log, hopefully it is fine.
```

Comment:
As an example, starting keycloak with `--spi-cache-embedded-default-node-name=my-node --spi-cache-embedded-default-site-name=my-site --spi-cache-embedded-default-machine-name=my-machine --spi-cache-embedded-default-rack-name=my-rack` will log the following

`ISPN000094: Received new cluster view for channel ISPN: [my-node(site-id=my-site, rack-id=my-rack, machine-id=my-machine)|0] (1) [my-node(site-id=my-site, rack-id=my-rack, machine-id=my-machine)]`

Reply:
The second argument is void, as the IP address is included in the hostname of the worker node. 
```
* Format for an instance in us-east-1: private-ipv4-address.ec2.internal
Example: ip-10-24-34-0.ec2.internal
* Format for an instance in any other AWS Region: private-ipv4-address.region.compute.internal
Example: ip-10-24-34-0.us-west-2.compute.internal
```
Honestly, I don't see any benefit for debugging either, but I'll change to the node name.

Reply:
Ok I didn't realise that, but I guess that's implementation independent. I don't have strong views on this either way in that case.